src/model/document.js

```diff
@@ -159,7 +162,10 @@ export default class Document {
 
 		this.version++;
 
-		this.history.addOperation( operation );
+		if ( operation.delta ) {
+			// Right now I can't imagine operations without deltas, but let's be safe.
```

Comment:
You can hide an error this way. Silent exit is never a good solution. If you want to be save throw error.